pkg/cmd/cli/cmd/rsh.go

```diff
@@ -143,5 +144,15 @@ func (o *RshOptions) Validate() error {
 
 // Run starts a remote shell session on the server
 func (o *RshOptions) Run() error {
+	// Insert the TERM into the command to be run
+	originalCommand := o.Command
+	o.Command = []string{"/bin/sh", "-c", fmt.Sprintf("TERM=vt100 %s", strings.Join(o.Command, " "))}
```

Comment:
also, we can't assume that joining args with spaces is correct... this doesn't handle quoting/escaping